xml/System.Data.Spatial/DbGeometry.xml

```diff
@@ -475,8 +475,8 @@
         <ReturnType>System.Nullable&lt;System.Double&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the elevation (Z coordinate) of this DbGeometry value, if it represents a point. &lt;returns&gt;The elevation (Z coordinate) of this geometry value, if it represents a point; otherwise <see langword="null" />.&lt;/returns&gt;</summary>
-        <value>The elevation (Z coordinate) of this DbGeometry value.</value>
+        <summary>Gets the elevation (Z coordinate) of this DbGeometry value, if it represents a point.</summary>
+        <value>The elevation (Z coordinate) of this geometry value, if it represents a point; otherwise <see langword="null" />.</value>
```

Comment:
```suggestion
        <value>The elevation (Z coordinate) of this geometry value, if it represents a point; otherwise, <see langword="null" />.</value>
```